tests/testcases/core/db_classes/EE_CPT_Base_Test.php

```diff
@@ -118,6 +118,24 @@ public function testGetPrettyCurrentPostRemainingTheSame()
             "'{$event_with_reg_desc}' should not be the same as '{$other_event_desc}'"
         );
     }
+
+
+    /**
+     * @ticket 418 https://github.com/eventespresso/event-espresso-core/issues/418
+     */
+    public function testAddEventCategory()
+    {
+        $e = $this->new_model_obj_with_dependencies('Event');
+        /**
+         * @var $e EE_Event
+         */
+        $ee_term_taxonomy = $e->add_event_category(
+            'Rebel Ships',
+            'Events look at important Rebel Space Ships'
+        );
+        $this->assertTrue($ee_term_taxonomy instanceof EE_Term_Taxonomy);
+        $this->assertNotEquals(0, $ee_term_taxonomy);
```

Comment:
Not sure both these assertions are necessary?  All that needs to be asserted is just the first right?

Reply:
oh oups, the second one should be testing the count